src/bake/DevServer.zig

```diff
@@ -2034,15 +2045,16 @@ pub fn IncrementalGraph(side: bake.Side) type {
             const dev = g.owner();
             dev.graph_safety_lock.assertLocked();
 
-            const abs_path = ctx.sources[index.get()].path.text;
+            const path = ctx.sources[index.get()].path;
```

Comment:
Is this copy intentional? Can it be a pointer instead? `Path` is not a small struct
```suggestion
            const path = &ctx.sources[index.get()].path;
```